ONBOARDING.md

```diff
@@ -3,67 +3,41 @@
 This document is an outline of the things we tell new Build WG members at their
 onboarding session.
 
-### Before the onboarding session
+## Onboarding checklists
 
-* Email the new member and obtain their public GPG key
-* Add them to the `test` group in `nodejs-private/secrets`
-* Add them to the `nodejs/build` and `nodejs-private/build` GitHub teams
-* Ask them to start looking over `services.md` and `jenkins-guide.md`,
-  and to try and have any questions ready
-* Ask the onboardee to install:
-  * Ruby and the `dotgpg` gem
-  * Python 2.7 and Ansible
+### Onboarding to build-test issue checklist
 
-### General information
-
-* Responsibilities: The Build WG manages everything in
-  [`doc/services.md`](doc/services.md)
-* Members/"who does what"/interest areas
-* Technologies and software in use
+- [ ] Email the new member and obtain their public GPG key
+- [ ] Add them to the `test` group in `nodejs-private/secrets`
+- [ ] Add them to the `nodejs/build` and `nodejs-private/build` GitHub teams
+- [ ] Schedule a meeting with the member to:
+    - [ ] Walk them through the infrastructure and what other members do 
+    - [ ] Explain how to decrypt nodejs/secrets 
+    - [ ] Practice running the `jenkins/worker/create.yml` playbook
+  on one of the machines in the test CI cluster
+    - [ ] Answer any questions they may have
+- [ ] Before the meeting the Onboardee is to:
```

Comment:
move the "before the meeting" to before the checklist of things to do in the meeting?